### DISCUSSION TITLE
Can we use Sass @use over @import?

### DISCUSSION THREAD

**Discussion by razor-x** [seamapi/react#81] [Q&A] [answered]
> The Sass team discourages the continued use of the @import rule. Sass will [gradually phase it out](https://github.com/sass/sass/blob/master/accepted/module-system.md#timeline) over the next few years, and eventually remove it from the language entirely. Prefer the [@use rule](https://sass-lang.com/documentation/at-rules/use) instead. (Note that only Dart Sass currently supports @use. Users of other implementations must use the @import rule instead.)

> The Sass migrator automatically updates your Sass files to help you move on to the latest and greatest version of the language. Each of its commands migrates a single feature, to give you as much control as possible over what you update and when.

https://sass-lang.com/documentation/cli/migrator

**Comment by razor-x** [comment]
@mikewuu Moving from https://github.com/seamapi/react/pull/47/files#r1184382097

**Reply by mikewuu** [reply, marked as answer]
I think we can/should. Created issue: https://github.com/seamapi/react/issues/84
